opg-phpcs-1604/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.service.opg.digital/opg-php-fpm-1604
+
+RUN apt-get update && apt-get install -y \
```

Comment:
these packages should be available from the base fpm image so there should be no apt commands in the derivative containers, unless they are for unique requirements